test/sol6/kyberNetwork.js

```diff
@@ -1752,6 +1873,88 @@ contract('KyberNetwork', function(accounts) {
+
+            // first rebate wallet rebate entitled
+            let rebateWallet = rebateWallets[0];
+            let actualBalance = await feeHandler.rebatePerWallet(rebateWallet);
+            Helper.assertGreater(actualBalance, rebateWalletBalance0[rebateWallet]);
+            
+            // second rebate wallet not rebate entitled
+            rebateWallet = rebateWallets[1];
+            actualBalance = await feeHandler.rebatePerWallet(rebateWallet);
+            Helper.assertEqual(actualBalance, rebateWalletBalance0[rebateWallet]);
+        });
+
+        it("should not have any fees if fee accounted data set to false", async() => {
+            // set fee accounted data to false
+            await storage.setFeeAccountedPerReserveType(false, false, false, false, false, false, {from: admin});
+            await storage.setEntitledRebatePerReserveType(false, false, false, false, false, false, {from: admin});
+            
+            let payoutBalance0 = await feeHandler.totalPayoutBalance();
+            let srcQty = oneEth;
+
+            await network.tradeWithHintAndFee(networkProxy, ethAddress, srcQty, srcToken.address, taker,
+                maxDestAmt, minConversionRate, platformWallet, 0, '0x', {from: networkProxy, value: srcQty});
+            let payoutBalance1 = await feeHandler.totalPayoutBalance();
+            Helper.assertEqual(payoutBalance0, payoutBalance1);
+
+            // reset fees
+            await storage.setFeeAccountedPerReserveType(true, true, true, false, true, true, { from: admin });
+            await storage.setEntitledRebatePerReserveType(true, false, true, false, true, true, {from: admin});
+        });
+
+        it("should not have any fees if entitled rebate data is set to false", async() => {
+            await storage.setFeeAccountedPerReserveType(true, true, true, true, true, true, {from: admin});
+            // set entitled rebate data to false
+            await storage.setEntitledRebatePerReserveType(false, false, false, false, false, false, {from: admin});
+            
+            let payoutBalance0 = await feeHandler.totalPayoutBalance();
+            let srcQty = oneEth;
+
+            await network.tradeWithHintAndFee(networkProxy, ethAddress, srcQty, srcToken.address, taker,
+                maxDestAmt, minConversionRate, platformWallet, 0, '0x', {from: networkProxy, value: srcQty});
+            let payoutBalance1 = await feeHandler.totalPayoutBalance();
+            Helper.assertEqual(payoutBalance0, payoutBalance1);
+
+            // reset fees
+            await storage.setFeeAccountedPerReserveType(true, true, true, false, true, true, { from: admin });
+            await storage.setEntitledRebatePerReserveType(true, false, true, false, true, true, {from: admin});
+        });
+
+        it("should not have fees only for platform wallet if fee accounted data set to false", async() => {
```

Comment:
I don't understand the title.
not have for platform wallet?
how is it related?

Reply:
Ah, wrong description. Fixed.